src/dedicated/overview/upgrading.md

```diff
@@ -0,0 +1,109 @@
+# Upgrading to the Integrated UI
+
+Older Platform.sh Dedicated projects (created prior to October 2017) used a separate Git repository for Production and Staging.  That also necessitated running most configuration changes through a ticket, and maintaining separate SSH credentials for each environment.
+
+These older projects can be upgraded to the new Integrated UI, which eliminates the extra Git repositories, many "must be a ticket" configuration changes, and makes the Production and Staging environments available in the UI.
+
+To add these environments to the Project Web Interface, review this entire document, complete a few preparatory steps, and submit a ticket. Your ticket is added to a queue for updating existing Dedicated projects.  The process may take time to complete, so check your ticket for details, timing, and other important information.
+
+We recommend this upgrade for all users.
+
+## New Features
+
+The new Project Web Interface provides the following features for the Pro plan Staging and Production environments:
+
+* Add and manage user access to the environments
+* Sync code between Staging and Production to Integration environments
+* Merge code from Integration environment to Staging environment to Production environment
+* Add and manage environment variables
+* Manage build and deploy hooks with the `.platform.app.yaml` file
+* Manage PHP versions and variables with the `.platform.app.yaml` file
+* Manage cron jobs with the `.platform.app.yaml` file
+* Configure environment settings
+* Access the environments using SSH and URL
+* View status, build logs, and deployment history
+
+You must still submit a support ticket to update and modify the following in the Staging and Production environments information:
+
+* Redirects from `routes.yaml` file
+* Managing PHP extensions
+* Managing mounts
+
+You cannot perform the following:
+
+* Branch from the Staging and Production environments
+* Synchronize data from the Staging and Production environments
+* Snapshot the Staging and Production environments
+
+### Branching hierarchy
+
+Before converting your project, the branches include a repository for Development, Staging, and Production. Each repository has a master branch with deployment targets configured for Staging and Production.
+
+After converting your project, the hierarchical relationships appear in your Project Web Interface with two, main environment branches for Staging, and Production:
+
+![Production and Staging are special children of master](/images/wings-branches.png)
```

Comment:
Broken image here, which I don't think got committed. Should probably go in `/images/dedicated` as well